### PR TITLE
Replace ColorEntry with ExprEntry in Color Sliders

### PR DESCRIPTION
**Pull request by Akylzhan**
Adds https://github.com/aseprite/aseprite/issues/3900

I agree that my contributions are licensed under the Individual Contributor License Agreement V4.0 ("CLA") as stated in https://github.com/igarastudio/cla/blob/main/cla.md

I have signed the CLA following the steps given in https://github.com/igarastudio/cla#signing

### CODE REVIEW

File: src/app/ui/color_sliders.cpp [hunk 1]
```diff
@@ -373,7 +373,7 @@ void ColorSliders::addSlider(const Channel channel,
   item.box       = new HBox();
   item.absSlider = new Slider(absMin, absMax, 0);
   item.relSlider = new Slider(relMin, relMax, 0);
-  item.entry     = new ColorEntry(item.absSlider, item.relSlider);
+  item.entry     = new ExprEntry();
```
Comment (dacap) [inline src/app/ui/color_sliders.cpp:376]: With this change we lost the behavior of `ColorEntry` (we cannot use up/down arrow keys to modify the value).

File: src/app/ui/color_sliders.h [hunk 2]
```diff
@@ -81,7 +82,7 @@ namespace app {
       ui::Box* box = nullptr;
       ui::Slider* absSlider = nullptr;
       ui::Slider* relSlider = nullptr;
-      ui::Entry* entry = nullptr;
+      app::ExprEntry* entry = nullptr;
```
Comment (dacap) [inline src/app/ui/color_sliders.h:85]: This change is not required.

**Review by dacap — CHANGES_REQUESTED** [review]
Hi @Akylzhan, thanks for your PR. Probably we require a different approach for this. Probably just changing:

```c++
class ColorEntry : public ExprEntry
```

**Comment by Akylzhan** [comment]
@dacap Yes, you are right. Did not think about that. Changed ColorEntry's base to ExprEntry

**Comment by dacap** [comment]
Thanks @Akylzhan, I've just merged this change in https://github.com/aseprite/aseprite/commit/3e171d0f6aebeff50138bce6fb98c987d1006723 with a missing setText("0") so the entry fields start with 0 in the *Edit > Adjustments > Hue/Saturation* dialog.

**Pull request closed without merging**